test/expect-tests/findlib_tests.ml

```diff
@@ -107,9 +107,8 @@ let%expect_test "configurator" =
 
 let%expect_test "builtins" =
   print_pkg_archives "str";
-  [%expect
-    {|
-    Hidden { byte = [ "stdlib/str.cma" ]; native = [ "stdlib/str.cmxa" ] } |}];
+  [%expect {|
+    Available { byte = []; native = [] } |}];
```

Comment:
I don't understand why this test result changed; can you explain it?

Reply:
Sure, the test demonstrates the bug being fixed. The OCAMLPATH directory in the test contains an `str` library. Previously, it was being completely ignored and pulled from the builtin map (see the stdlib/ prefix).

Now, it's being pulled from the OCAMLPATH directory as expected.

One more little thing to note is that `Hidden` was changed to `Available`. That's because the builtin `str` contains an `exists_if` that wasn't satisfied (see the fake paths). While the library doesn't have that.

Reply:
OK, thanks, but why are `byte` and `native` empty?

Reply:
It's just how the META file is defined for `str` in the tests.

Reply:
Right, thanks!